code/__DEFINES/id_cards.dm

```diff
@@ -0,0 +1,25 @@
+// The order in which this lists are defined matters. When attempting to add a wildcard access to a card,
+// the code will iterate over the list and stop at the first wildcard flag that can hold the access.
```

Comment:
> 
> 
> Isn't this counter to what's described in the design doc? Or am I misunderstanding the wording

This documentation is outdated - Adding a wildcard can take an optional "put this wildcard in this slot specifically" arg now, which is possible because I managed to nail some conditional formatting in the ID console tgui interface to make it sane.